pytorch_lightning/metrics/utils.py

```diff
@@ -15,7 +15,7 @@
 from typing import Optional
 
 import torch
-from deprecate import deprecated
+from deprecate import deprecated, void
```

Comment:
That's not really better here :D You only forwarded the import, but I think this void thing shouldn't live in the deprecate package in the first place, since this has nothing to do with deprecation.

Reply:
ok, copy-pasted the implementation :]